### DISCUSSION TITLE
Codespace Access Limited

### DISCUSSION THREAD

**Discussion by cuong-nn** [community/community#101930] [Codespaces]
### Select Topic Area

General

### Body

I'm trying to open a repository that was created by my college on Codespace and this is what shows up:

![image](https://github.com/community/community/assets/129546111/b0d5b094-a8ca-44d9-bb9f-ae513dbb1cd7)

Any input would be appreciated!

**Comment by n31t** [comment]
Hi.
To enable Codespaces for repositories, you need to contact the organization owner and ask them to enable Codespaces for those repositories.

Here are the steps on how to enable Codespaces for an organization:
1. Go to the organization's settings page.
2. Click on the "Codespaces" tab.
3. Under "General", make sure that the "Enable Codespaces" checkbox is selected.
4. If you want to pay for the usage of codespaces for members and collaborators, you can set a spending limit.
5. Click on the "Save" button.

**Reply by joshbla** [reply]
Thank you!!

**Comment by PwrSrg** [comment]
**Same issue here.**

<img width="427" height="252" alt="image" src="https://github.com/user-attachments/assets/bf0c4a9e-95b4-4645-bdd8-823ec68b4937" />

<br>But I do not see a "General" section in the Codespaces tab, nor do I see an "Enable Codespaces" checkbox.  Has it moved?

<img width="1363" height="2881" alt="screencapture-github-settings-codespaces-2025-08-07-14_03_45" src="https://github.com/user-attachments/assets/b7f410ba-1d9c-4f6d-b850-173ffcae8476" />

<br>Also, if Codespaces were _intentionally_ disabled wouldn't the message be something more like:

> Your organization has disabled Codespaces on this repository. Please contact your organization administrator for more information.

And if it is by design, why is it pointing me to **Github Support**?  I feel like there is more to this issue than is being said.
